src/lib/factories/dealer.ts

```diff
@@ -0,0 +1,18 @@
+import { SearchDealer } from "../../types/models/dealer"
+
+export function DealerPromotion(): SearchDealer {
```

Comment:
Would be nice if there is an option to override the defaults in case someone needs a custom promotion for a test case....

Reply:
ok

Reply:
done